### PR TITLE
Review congrats mobile

### PR DESCRIPTION
**Pull request by chrisdugne**
**Detailed purpose of the PR**

- using `atom.button` on slide and congrats
- added congrats view

<p>
Screenshot 2022-12-05 at 15 44 36
Screenshot 2022-12-05 at 15 43 03
</p>

### CODE REVIEW

**Comment by chrisdugne** [comment]
[[OR] Congrats - mobile](https://trello.com/c/E6WcHScz/2571-or-congrats-mobile)

File: packages/@coorpacademy-app-review/src/index.tsx [hunk 1]
```diff
@@ -3,7 +3,7 @@ import type {} from 'redux-thunk/extend-redux'; // https://github.com/reduxjs/re
 import {AnyAction, Store} from 'redux';
 import {useSelector, useDispatch, Provider} from 'react-redux';
 import AppReviewTemplate from '@coorpacademy/components/es/template/app-review';
-import {AppReviewProps} from '@coorpacademy/components/es/template/app-review/prop-types';
+import type {AppReviewProps} from '@coorpacademy/components/es/template/app-review/prop-types';
```
Comment (scalabw) [inline packages/@coorpacademy-app-review/src/index.tsx:6]: 😍

File: packages/@coorpacademy-components/src/organism/review-congrats/index.native.tsx [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+    'aria-label': ariaLabel,
+    animationLottie,
+    title,
+    cardCongratsStar,
+    cardCongratsRank,
+    buttonRevising,
+    buttonRevisingSkill
+  } = props;
+
+  const {theme} = useTemplateContext();
```
Comment (scalabw) [inline packages/@coorpacademy-components/src/organism/review-congrats/index.native.tsx:70]: can we type this by using "TemplateContextValues" ?

File: packages/@coorpacademy-components/src/organism/review-congrats/index.native.tsx [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+  }
+
+  const handleContinueRevisingPress = buttonRevising?.onClick || noop;
+  const handleReviseAnotherSkillPress = buttonRevisingSkill?.onClick || noop;
+
+  return (
+    <View style={styleSheet.congrats} accessibilityLabel={ariaLabel}>
+      <Text style={styleSheet.title}>{title}</Text>
+      <ScrollView showsHorizontalScrollIndicator={false} horizontal style={{height: 300}}>
+        {cardCongratsRank ? (
```
Comment (scalabw) [inline packages/@coorpacademy-components/src/organism/review-congrats/index.native.tsx:90]: can we use something like `cardCongratsRank && (<CardCongrats/>)`

File: packages/@coorpacademy-components/src/organism/review-congrats/index.native.tsx [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+          text={cardCongratsStar.reviewCardTitle}
+          value={cardCongratsStar.reviewCardValue}
+          iconColor={theme.colors.battle}
+          textColor={theme.colors.battle}
+          direction="row"
+        />
+      </ScrollView>
+
+      <View style={styleSheet.buttons}>
+        {buttonRevisingSkill ? (
```
Comment (scalabw) [inline packages/@coorpacademy-components/src/organism/review-congrats/index.native.tsx:113]: same

File: packages/@coorpacademy-components/src/organism/review-congrats/index.native.tsx [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+          <Button
+            onPress={handleReviseAnotherSkillPress}
+            accessibilityLabel={buttonRevisingSkill['aria-label']}
+            submitValue={buttonRevisingSkill.label}
+            style={{...BOX_STYLE, backgroundColor: '#fff'}}
+            textStyle={{color: '#123'}}
+          />
+        ) : null}
+        {buttonRevising ? (
```
Comment (scalabw) [inline packages/@coorpacademy-components/src/organism/review-congrats/index.native.tsx:122]: same

File: packages/@coorpacademy-components/src/organism/review-slide/index.native.tsx [hunk 6]
```diff
@@ -250,35 +201,35 @@ const Slide = (props: ReviewSlideProps) => {
...
+        answerUI={answerUI}
+        key="question-container"
+      />
+      <Button
+        disabled={validateButton.disabled}
+        submitValue={validateButton.label}
+        onPress={handleValidatePress}
+        testID={`slide-validate-button-${slideIndex}`}
+      />
+      {correctionPopinProps ? (
```
Comment (scalabw) [inline packages/@coorpacademy-components/src/organism/review-slide/index.native.tsx:224]: same as upper

File: packages/@coorpacademy-components/src/organism/review-slide/test/fixtures/initial-state.ts [hunk 7]
```diff
@@ -1,16 +1,19 @@
 import cloneDeep from 'lodash/fp/cloneDeep';
 import map from 'lodash/fp/map';
 import set from 'lodash/fp/set';
+import {Answer} from '../../../../molecule/answer/prop-types';
```
Comment (scalabw) [inline packages/@coorpacademy-components/src/organism/review-slide/test/fixtures/initial-state.ts:4]: `import type {Answer} from '../../../../molecule/answer/prop-types';` ?

File: packages/@coorpacademy-components/src/molecule/card-congrats/index.native.tsx [hunk 8]
```diff
@@ -0,0 +1,99 @@
...
+    // eslint-disable-next-line @coorpacademy/coorpacademy/no-overwriting-spread
+    card: {
+      alignItems: 'center',
+      paddingTop: 32,
+      paddingBottom: 24,
+      borderRadius: 24,
+      margin: 10,
+      width: 280,
+      height: 317,
+      ...BOX_STYLE,
```
[the remark this answers is not in the document]
Comment (chrisdugne) [reply packages/@coorpacademy-components/src/molecule/card-congrats/index.native.tsx:35]: BOX_STYLE allows to factorise the shadow stuffs,
here I still need to override background

File: packages/@coorpacademy-components/src/organism/review-slide/test/fixtures/initial-state.ts (hunk 7 above)
Comment (chrisdugne) [reply packages/@coorpacademy-components/src/organism/review-slide/test/fixtures/initial-state.ts:4]: 👍 https://github.com/CoorpAcademy/components/commit/3897d3810ba1147bcea6f13880bbe7e790014042

**Pull request merged**